framework/src/main/java/org/checkerframework/framework/util/Contract.java

```diff
@@ -7,12 +7,12 @@
 import org.checkerframework.framework.qual.ConditionalPostconditionAnnotation;
 import org.checkerframework.framework.qual.EnsuresQualifier;
 import org.checkerframework.framework.qual.EnsuresQualifierIf;
-import org.checkerframework.framework.qual.EnsuresQualifiers;
-import org.checkerframework.framework.qual.EnsuresQualifiersIf;
 import org.checkerframework.framework.qual.PostconditionAnnotation;
 import org.checkerframework.framework.qual.PreconditionAnnotation;
 import org.checkerframework.framework.qual.RequiresQualifier;
-import org.checkerframework.framework.qual.RequiresQualifiers;
+import org.checkerframework.framework.util.Contract.ConditionalPostcondition;
```

Comment:
From the diffs in this file it seems a bit odd why these new imports are required.
Do you see the reason?